dist/index.html

```diff
@@ -78,6 +79,46 @@ <h2 id="availableText">Rooms Available Today</h2>
       </section>
     </section>
   </main>
+  <section id="managerDash" class="manager hidden">
```

Comment:
Nice work around for displaying manager dashboard as opposed to customer dashboard!